app/components/c-linklist.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <ul :class="b('')">
+    <li v-for="(value, index) in items" :class="b('item')" :key="index">
+      <a :class="b('link')"
+         :href="value"
+         target="_blank"
```

Comment:
- [ ] Needs a title attribute (if not delivered from backend, add something like "PDF Download"